src/partials/layout.html

```diff
@@ -188,6 +189,43 @@
           </span>
         </div>
       </div>
+    </section> -->
+
+    <section id="sponsors" class="bg-gray-800">
+      <div
+        class="m-auto max-w-5xl px-4 py-16 flex flex-col sm:flex-row gap-20 sm:gap-0 items-center justify-around"
+      >
+        <!-- Sponsors-->
+        <div class="flex flex-col items-center sm:items-start">
+          <span class="text-lg font-semibold tracking-widest text-white">
+            SPONSORS
+          </span>
+          <a class="flex justify-center mt-4" href="https://fly.io/">
+            <img
+              src="/images/flyio_logo_with_text.svg"
+              alt="Flyio Logo"
+              width="250"
+            />
+          </a>
+        </div>
+
+        <!-- Supporters -->
+        <div class="flex flex-col items-center sm:items-start">
+          <span class="text-base tracking-widest text-white"> SUPPORTERS </span>
+
+          <!-- Hugging Face -->
+          <div class="flex items-center mt-4">
```

Comment:
Hugging Face doesn't have a link to Hugging Face's website, but Fly.io does. Is that on purpose because one is a sponsor and the other is a supporter?

Or we just missed to add a link here?